content/riak/ts/1.3.1/installing/aws.md

```diff
@@ -12,8 +12,7 @@ project_version: "1.3.1"
 toc: true
 version_history:
   locations:
-    - ["1.3.0-1.3.1", "installing/aws"]
-    - ["1.4.0+",      "setup/installing/aws"]
+  in: "1.3.1+"
```

Comment:
Why remove the 1.3.0? And why remove the locations for the 1.4.0+

Reply:
Might have made a mistake here. I'll revert this one and the one below